### ISSUE TITLE
high cpu load on Lana's Bridge

### ISSUE THREAD

**Issue by mithrand0** [ReactiveDrop/reactivedrop_public_src#356]
The cpu load on Lana's Bridge is extremely high compared to other maps, often hitting 100% on higher difficulties with full lobbies. It would be good to check if this map can be optimized.

**Comment by jhh8** [comment]
did you check on beta? i did some changes in #334

**Comment by jhh8** [comment]
still wondering if on beta this is fine or need mmore changes

**Comment by mithrand0** [comment]
yes, still the same fps drops to single digits due very high cpu load

**Comment by wesaamt** [comment]
Hi
